corehq/blobs/migratingdb.py

```diff
@@ -45,3 +45,7 @@ def get_path(self, *args, **kw):
 
     def copy_blob(self, content, info, bucket):
         self.new_db.copy_blob(content, info, bucket)
+
+    def close(self):
```

Comment:
@millerdev Is there any reason the migrating db didn't have a close method? I added it so that I could change the tests around to properly call `setUpClass` and `tearDownClass`